README.md

```diff
@@ -27,6 +27,36 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### Debian Repository
+
+- Save the following as `/etc/apt/sources.list.d/etcher.list`:
+
+```
+deb https://dl.bintray.com/resin-io/debian stable etcher
```

Comment:
I'd prefer to keep this way, at for now, since we're using code fences throughout the same repository. I'm happy to revise this decision for all documentation if there is any particular benefit on doing so.